better-font-awesome.php

```diff
@@ -268,8 +301,8 @@ private function initialize_better_font_awesome_library( $options ) {
         
         $args = array(
             'version'             => isset( $options['version'] ) ? $options['version'] : $this->option_defaults['version'],
-            'minified'            => isset( $options['minified'] ) ? $options['minified'] : '',
-            'remove_existing_fa'  => isset( $options['remove_existing_fa'] ) ? $options['remove_existing_fa'] :'',
+            'minified'            => isset( $options['minified'] ) ? $options['minified'] : $this->option_defaults['minified'],
+            'remove_existing_fa'  => isset( $options['remove_existing_fa'] ) ? $options['remove_existing_fa'] : $this->option_defaults['remove_existing_fa'],
```

Comment:
I find ternary operators look a bit cleaner and easier to read if you align on ```?``` and ```: ```as well